src/sonic-host-services/scripts/hostcfgd

```diff
@@ -57,15 +71,188 @@ def obfuscate(data):
         return data
 
 
-def run_cmd(cmd, log_err = True):
-    try:
-        subprocess.check_call(cmd, shell = True)
-    except Exception as err:
-        if log_err:
-            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
-                  .format(err.cmd, err.returncode, err.output))
-        return err.returncode
-    return 0
+class Feature(object):
+    """ Represents a feature configuration from CONFIG DB data. """
```

Comment:
Do we need use `CONFIG_DB` to replace `CONFIG DB`?

Reply:
Ok